README.md

```diff
@@ -95,763 +93,54 @@ file](https://github.com/camptocamp/tilecloud-chain/blob/master/tilecloud_chain/
-
-    generate_tiles --cache <a_cache>
-
-And don't forget to generate the WMTS Capabilities:
-
-    generate_controller --capabilities
-
-### OpenLayers test page
-
-To generate a test page use:
-
-    generate_controller --openlayers
-
-Explain cost
-------------
-
-Configuration (default values):
-
-``` {.sourceCode .yaml}
-cost:
-    # [nb/month]
-    request_per_layers: 10000000
-    cloudfront:
-        download: 0.12,
-        get: 0.009
-    request_per_layers: 10000000
-    s3:
-        download: 0.12,
-        get: 0.01,
-        put: 0.01,
-        storage: 0.125
-    sqs:
-        request: 0.01
-```
-
-Layer configuration (default values):
-
-``` {.sourceCode .yaml}
-cost:
-    metatile_generation_time: 30.0,
-    tile_generation_time: 30.0,
-    tile_size: 20.0,
-    tileonly_generation_time: 60.0
-```
-
-The following commands can be used to know the time and cost to do generation:
-
-    generate_controller --cost
-
-### Useful options
-
-`--quiet` or `-q`: used to display only errors.
-
-`--verbose` or `-v`: used to display info messages.
-
-`--debug` or `-d`: used to display debug message, please use this option to report issue. With the debug mode
-we don't catch exceptions, and we don't log time messages.
-
-`--test <n>` or `-t <n>`: used to generate only `<n>` tiles, useful for test.
-
-The logging format is configurable in the`config.yaml` - `generation/log_format`,
-[See](http://docs.python.org/2/library/logging.html#logrecord-attributes).
-
-Important remarks
------------------
-
-Especially on S3 the grid name, the layer name, the dimensions, can't be changed (understand if we want to
-change them we should regenerate all the tiles).
-
-By default we also can't insert a zoom level, if you think that you need it we can set the grid property
-`matrix_identifier: resolution`, bit it don't work with MapCache.
-
-Please use the `--debug` to report issue.
-
-From sources
-------------
+## From sources
 
 Build it:
 
-``` {.sourceCode .bash}
+```{.sourceCode .bash}
 git submodule update --recursive
 python3 -m venv .build/venv
 .build/venv/bin/pip install -r requirements.txt
 .build/venv/bin/pip install -e .
 .build/venv/bin/pip install -r dev-requirements.txt
 ```
 
-Run prospector
---------------
+## Run prospector
 
-``` {.sourceCode .bash}
+```{.sourceCode .bash}
 .build/venv/bin/prospector
 ```
 
-Run the tests
--------------
+## Run the tests
 
 Setup your environment:
 
-``` {.sourceCode .bash}
+```{.sourceCode .bash}
 touch tilecloud_chain/OpenLayers.js
 docker build --tag camptocamp/tilecloud-chain .
 docker-compose -p tilecloud up
 ```
 
 To run the tests:
 
-``` {.sourceCode .bash}
+```{.sourceCode .bash}
 docker-compose -p tilecloud exec test python setup.py nosetests --logging-filter=tilecloud,tilecloud_chain --attr '!'nopy3
 ```
 
-Environment variables
----------------------
+## Documentation
+
+As documatation you can read the `tilecloud_chain/USAGE.rst`.
```

Comment:
May use absolute link, not sure a relative link will be correctly expanded in PyPi ?

Reply:
+1 :-)